processing/src/main/java/org/apache/druid/query/QueryContexts.java

```diff
@@ -188,22 +195,22 @@ public static <T> boolean isUseResultLevelCache(Query<T> query)
 
   public static <T> boolean isUseResultLevelCache(Query<T> query, boolean defaultValue)
   {
-    return parseBoolean(query, "useResultLevelCache", defaultValue);
+    return parseBoolean(query, USE_RESULT_LEVEL_CACHE_KEY, defaultValue);
   }
-
   public static <T> boolean isFinalize(Query<T> query, boolean defaultValue)
+
   {
     return parseBoolean(query, FINALIZE_KEY, defaultValue);
   }
 
   public static <T> boolean isSerializeDateTimeAsLong(Query<T> query, boolean defaultValue)
   {
-    return parseBoolean(query, "serializeDateTimeAsLong", defaultValue);
+    return parseBoolean(query, SERIALIZE_DATE_TIME_AS_LONG_KEY, defaultValue);
   }
 
   public static <T> boolean isSerializeDateTimeAsLongInner(Query<T> query, boolean defaultValue)
   {
-    return parseBoolean(query, "serializeDateTimeAsLongInner", defaultValue);
+    return parseBoolean(query, SERIALIZE_DATE_TIME_AS_LONG_INNER_KEY, defaultValue);
```

Comment:
Is there any reason for `parseBoolean` and `getAsBoolean` to remain different? Now that we have `getAsBoolean`, shall it replace `parseBoolean`?

(Btw, is the behavior the same? I haven't checked)

Reply:
Good point. I didn't really pay attention to the `parseFoo` methods. Those methods appear to only handle the value-as-string case, where as the `getAsFoo` method handle maps with values of the expected type. Went ahead and changed the `parseFoo` methods to use `getAsFoo` so we're consistent.